[PATCH] fix build issue in CArchDaemonUnix

exit() needs #include <stdlib.h> for the prototype.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 lib/arch/CArchDaemonUnix.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: synergy-1.3.1/lib/arch/CArchDaemonUnix.cpp
===================================================================
--- synergy-1.3.1.orig/lib/arch/CArchDaemonUnix.cpp
+++ synergy-1.3.1/lib/arch/CArchDaemonUnix.cpp
@@ -19,6 +19,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <errno.h>
+#include <stdlib.h>
 
 //
 // CArchDaemonUnix
